File: pmd/java/listas/listaex01/ex2/Conta.java
package listas.listaex01.ex2;

import java.util.Scanner;

public class Conta {

  private int numero;
  private double saldo;

  public Conta(int numero, double saldo) {
    this.numero = numero;
    this.saldo = saldo;
  }

  public int getNumero() {
    return numero;
  }

  public double getSaldo() {
    return saldo;
  }

  public void setNumero(int numero) {
    this.numero = numero;
  }

  public void setSaldo(double saldo) {
    this.saldo = saldo;
  }

}

File: pmd/python/listas/listaex01/ex2/conta.py
class Conta:

    def __init__(self, numero=0, saldo=0.0):
        self.__numero = numero 
        self.__saldo = saldo

    def numero(self):
        return self.__numero

    def saldo(self):
        return self.__saldo

    def depositar(self, valor):
        self.__saldo += valor

    def sacar(self, valor):
        if valor > self.__saldo: 
            return False
        else: 
            self.__saldo -= valor
            return True
